docs/Developers/Newcomers_Guide.rst

```diff
@@ -336,9 +336,11 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
 .. note::
 
-    **Do not only fix a newcomer issue!** Supervising newcomers is really a lot
-    of work. We're all volunteers and we can't keep this up if you don't help
-    us in other areas as well!
+    **Do not only fix a newcomer issue!** It is highly recommmended that you
+    fix one newcommer issue to get familiar with the workflow at coala and
+    then proceed to a difficulty/low issue with that it is very important for
+    you to start reviweing difficulty/low or newcomer issue as it very
+    essential in stage of learning.
```

Comment:
Maybe this could be split into two sentences? I think it should be specified that if one is familiar with open source one can skip the newcomer issues altogether.